Changelog.md

```diff
@@ -23,7 +21,6 @@ Bugfixes:
  * AST: Export ``canonicalName`` for ``UserDefinedValueTypeDefinition`` and ``ContractDefinition``.
 
 
-
```

Comment:
Some cleaning up needed.

Reply:
This is weird, I don't have it in my local diff